Index: src/com/joulespersecond/seattlebusbot/FindActivity.java
===================================================================
--- src/com/joulespersecond/seattlebusbot/FindActivity.java	(revision 166)
+++ src/com/joulespersecond/seattlebusbot/FindActivity.java	(working copy)
@@ -184,10 +184,14 @@
     protected final void setResponse(ObaResponse response) {
         mResponse = response;
         TextView empty = (TextView) findViewById(android.R.id.empty);
-        if (response.getCode() == ObaApi.OBA_OK) {
+        final int code = response.getCode();
+        if (code == ObaApi.OBA_OK) {
             empty.setText(R.string.find_hint_noresults);
             setResultsAdapter(response);
         }
+        else if (code == ObaApi.OBA_NOT_FOUND) {
+            empty.setText(R.string.find_hint_noresults);
+        }
         else {
             empty.setText(R.string.generic_comm_error);
         }  
Index: src/com/joulespersecond/oba/ObaApi.java
===================================================================
--- src/com/joulespersecond/oba/ObaApi.java	(revision 166)
+++ src/com/joulespersecond/oba/ObaApi.java	(working copy)
@@ -24,7 +24,7 @@
     
     private static final String API_KEY = "v1_BktoDJ2gJlu6nLM6LsT9H8IUbWc=cGF1bGN3YXR0c0BnbWFpbC5jb20=";
     // NOTE: This could be provided by Settings to use different versions of the server.
-    private static final String OBA_URL = "http://api.onebusaway.org/api/where";
+    private static final String OBA_URL = "http://soak-api.onebusaway.org/api/where";
     
     public static final double E6 = 1000*1000;
     
